continuous_integration/install.sh

```diff
@@ -40,25 +40,48 @@ if [[ "$DISTRIB" == "conda" ]]; then
         # Make sure that MKL is not used
         conda remove --yes --features mkl || echo "MKL not installed"
     fi
+    # Install cython
+    conda install --yes cython
 
 elif [[ "$DISTRIB" == "ubuntu" ]]; then
     # At the time of writing numpy 1.9.1 is included in the travis
-    # virtualenv but we want to used numpy installed through apt-get
+    # virtualenv but we want to used nNumpy installed through apt-get
     # install.
     deactivate
     # Create a new virtualenv using system site packages for numpy and scipy
     virtualenv --system-site-packages testvenv
     source testvenv/bin/activate
     pip install nose
+    # install cython using astropy wheels (no wheels on PyPi)
+    pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
 fi
 
 if [[ "$COVERAGE" == "true" ]]; then
     pip install coverage coveralls
 fi
 
+TRAVIS_REPO=$(pwd)
```

Comment:
Is it the travis directory? It's not clear what it is given that latter you either pull or clone this variable.